README.md

```diff
@@ -1,99 +1,62 @@
-# Mesosphere Universe [![Build Status](https://teamcity.mesosphere.io/guestAuth/app/rest/builds/buildType:(id:Oss_Universe_Ci)/statusIcon)](https://teamcity.mesosphere.io/viewType.html?buildTypeId=Oss_Universe_Ci&guest=1)
-
-The DCOS package repository for packages that have been certified by Mesosphere.
-
-## Installation
-
-The latest [DC/OS](https://mesosphere.com/product/) comes pre-configured to use the Universe
-repository.
-
-If you would like to add this to your DC/OS manually:
-
-```
-dcos package repo add Universe https://universe.mesosphere.com/repo
+# Mesosphere Universe
+
+| Build | Status |
+|---|---|
+|CI   | [![Build Status](https://teamcity.mesosphere.io/guestAuth/app/rest/builds/buildType:(id:Oss_Universe_Ci)/statusIcon)](https://teamcity.mesosphere.io/viewType.html?buildTypeId=Oss_Universe_Ci&guest=1)|
+| Universe Server | [![Build Status](https://teamcity.mesosphere.io/guestAuth/app/rest/builds/buildType:(id:Oss_Universe_UniverseServer)/statusIcon)](https://teamcity.mesosphere.io/viewType.html?buildTypeId=Oss_Universe_UniverseServer&guest=1)|
+Mesosphere Universe registry of packages made available for DC/OS Clusters.
+
+#### Table of Contents
+* [Universe Function](#universe-function)
+* [Publish a Package](#publish-a-package-1)
+  * [Creating a Package](#creating-a-package)
+    * [`package.json`](#packagejson)
+      * [`.minDcosReleaseVersion`](#mindcosreleaseversion)
+    * [`config.json`](#configjson)
+    * [`marathon.json.mustache`](#marathonjsonmustache)
+    * [`command.json`](#commandjson)
+    * [`resource.json`](#resourcejson)
+      * [Docker Images](#docker-images)
+      * [Images](#images)
+      * [CLI Resources](#cli-resources)
+  * [Submit your Package](#submit-your-package)
+* [Repository Consumption](#repository-consumption-1)
+  * [Universe Server](#universe-server)
+    * [Build Universe Server locally](#build-universe-server-locally)
+    * [Run Universe Server](#run-universe-server)
+  * [Consumption Protocol](#consumption-protocol)
+  * [Supported DC/OS Versions](#supported-dcos-versions)
+
+## Universe Function
+This git repo facilitates three necessary functions to publish, store and consume packages.
+
+### Publish a Package
+
+To publish a package to Universe, fork this repo and open a Pull Request. A set of automated builds will be ran against
+the Pull Request to ensure the modifications made in the PR conform to the schema of the repository and each of the
+files corresponding to your package.  See [Creating a Package](#creating-a-package) for details.
+
+### Registry of Packages
+
+The registry of published packages is maintained as the contents of this repo in the `repo/packages` directory. As of
+repository version `3.0` multiple packaging versions are allowed to co-exist in the same repository. Validation of
+packages are coordinated for the version of the package and are required to be adhered to.
```

Comment:
"Validation of packages are coordinated for the version of the package" -- this is saying that packages are validated with respect to their packaging version?